internal/cli/cmd/clusterdefinition/describe.go

```diff
@@ -0,0 +1,174 @@
+	}
+	cmd := &cobra.Command{
+		Use:               "describe",
+		Short:             "Describe ClusterDefinition.",
+		Example:           describeExample,
+		Aliases:           []string{"desc"},
+		ValidArgsFunction: util.ResourceNameCompletionFunc(f, types.ClusterDefGVR()),
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(o.complete(args))
+			cmdutil.CheckErr(o.run())
+		},
+	}
+	return cmd
+}
+
+func (o *describeOptions) complete(args []string) error {
+	var err error
+
+	if len(args) == 0 {
+		return fmt.Errorf("cluster definition name should be specified")
+	}
+	o.names = args
+
+	if o.client, err = o.factory.KubernetesClientSet(); err != nil {
+		return err
+	}
+
+	if o.dynamic, err = o.factory.DynamicClient(); err != nil {
+		return err
+	}
+
+	if o.namespace, _, err = o.factory.ToRawKubeConfigLoader().Namespace(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *describeOptions) run() error {
+	for _, name := range o.names {
+		if err := o.describeClusterDef(name); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *describeOptions) describeClusterDef(name string) error {
+	// get cluster definition
+	clusterDefObject, err := o.dynamic.Resource(types.ClusterDefGVR()).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	clusterDef := v1alpha1.ClusterDefinition{}
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(clusterDefObject.Object, &clusterDef); err != nil {
+		return err
+	}
+
+	// get backup policy templates of the cluster definition
+	opts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=%s", constant.ClusterDefLabelKey, name),
+	}
+	backupTemplatesListObj, err := o.dynamic.Resource(types.BackupPolicyTemplateGVR()).List(context.TODO(), opts)
+	if err != nil {
+		return err
+	}
+	var backupPolicyTemplates []*v1alpha1.BackupPolicyTemplate
+	for _, item := range backupTemplatesListObj.Items {
+		backupTemplate := v1alpha1.BackupPolicyTemplate{}
+		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(item.Object, &backupTemplate); err != nil {
+			return err
+		}
+		backupPolicyTemplates = append(backupPolicyTemplates, &backupTemplate)
+	}
+
+	showClusterDef(&clusterDef, o.Out)
+
+	showBackupConfig(backupPolicyTemplates, o.Out)
+
+	return nil
+}
+
+func showClusterDef(cd *v1alpha1.ClusterDefinition, out io.Writer) {
+	if cd == nil {
+		return
+	}
+	fmt.Fprintf(out, "Name: %s\t Type: %s\n\n", cd.Name, cd.Spec.Type)
+}
+
+func showBackupConfig(backupPolicyTemplates []*v1alpha1.BackupPolicyTemplate, out io.Writer) {
+	if len(backupPolicyTemplates) == 0 {
+		return
+	}
+	fmt.Fprintf(out, "Backup Config:\n")
+	tbl := printer.NewTablePrinter(out)
+	tbl.SetHeader("AUTO-BACKUP", "BACKUP-SCHEDULE", "BACKUP-METHOD", "BACKUP-RETENTION")
+	defaultBackupPolicyTemplate := &v1alpha1.BackupPolicyTemplate{}
+	for _, item := range backupPolicyTemplates {
+		if item.Annotations[dptypes.DefaultBackupPolicyTemplateAnnotationKey] == "true" {
```

Comment:
Sometimes the bpt(backupPolicyTemplate) may not have a default annotation. If there is only one bpt, it will be directly used without determining whether it is default or not. 
Additionally, the main purpose of this feature is to output the backup methods for users to check which backup methods are available when creating a cluster, so that they can specify it when create cluster. The controller will then create corresponding schedules based on the backup information in the cluster CR. The schedules in bpt only provide some default values for the cluster backup schedules.